### PR TITLE
new API with useMutableSource

### PR DESCRIPTION
**Pull request by dai-shi**
API is not based on Context. It's optional.

Check list:
- [x] implementation
- [x] readme

### CODE REVIEW

**Comment by dai-shi** [comment]
Published: https://www.npmjs.com/package/reactive-react-redux/v/5.0.0-alpha.1

**Comment by dai-shi** [comment]
Published: https://www.npmjs.com/package/reactive-react-redux/v/5.0.0-alpha.2

File: README.md [hunk 1]
```diff
@@ -81,11 +83,11 @@ const reducer = (state = initialState, action) => {
   }
 };
 
-const store = createStore(reducer);
+const store = patchStoroe(createStore(reducer));
```
Comment (azmenak) [inline README.md]: Typo 😄 
```suggestion
const store = patchStore(createStore(reducer));
```

Comment (dai-shi) [reply README.md]: Thanks 😄

**Comment by faceyspacey** [comment]
I implemented it as documented with the react-redux Context recipe, and it works (`useTrackedState`), but I get this error in the console after state changes:

```
index.js?4ca0:22 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in TradeRow (created by TradeList)
    in div (created by TradeList)
    in div (created by Component)
    in Component (created by TradeList)
    in TradeList (created by Switcher)
```

**Comment by dai-shi** [comment]
@faceyspacey Hmm, is it possible for you to make a small reproduction in codesandbox?
Maybe from this working version? https://codesandbox.io/s/magical-mahavira-p22qv (or another simpler example?)

**Comment by faceyspacey** [comment]
I will at some point, but I'm using a custom setup with a babel plugin that makes `MyComponent = (props, state, actions) => ...` possible. `state` here FYI contains both reducers *and selectors*. 

I've been meaning to show you how I'm doing it by making a small repro of this aspect of the system, so we can figure out how to memoize selectors across usages, as selectors in Respond are defined once upfront (not in component functions). I'll make the repro when I get to this point.

If it's not occurring for you, it's likely due to my system. I have an idea of what will be causing. I'll make my system more normal, and see if that removes the issue.

**Comment by dai-shi** [comment]
> If it's not occurring for you, it's likely due to my system.

Not for me. Most likely, it's from somewhere around `dispatch` in your system.
I'm not 100% sure, though. `useMutableSource` is still new to us.

**Comment by dai-shi** [comment]
Published: https://www.npmjs.com/package/reactive-react-redux/v/5.0.0-alpha.3

**Comment by dai-shi** [comment]
https://www.npmjs.com/package/reactive-react-redux/v/5.0.0-alpha.4

**Comment by dai-shi** [comment]
https://www.npmjs.com/package/reactive-react-redux/v/5.0.0-alpha.5

**Comment by dai-shi** [comment]
Although useMutableSource hasn't landed, let me merge this PR and update README to indicate this is still an experimental project.

**Pull request merged**